README.md

```diff
@@ -39,26 +41,61 @@ itself and doesn't install in the current environment.
 
 Autohooks is a pure python library that installs a minimal
 [executable git hook](https://github.com/greenbone/autohooks/blob/master/autohooks/precommit/template).
-If autohooks isn't installed in your current python path the hooks aren't
-executed. So autohooks is always opt-in by installing the package into your
-current development environment. It would be even possible to run different
-versions of autohooks by switching the environment.
+It allows you to decide how to maintain your hook dependencies by supporting
+different modes.
 
-Autohooks doesn't interfere with your work. If autohooks can't be run or fails
-executing a plugin, an error is shown only and the git hook will proceed.
+## Modes
+
+Currently two modes for using autohooks are supported:
+
+* `pythonpath` and
+* `pipenv`
+
+The modes handle how autohooks, the plugins and their dependencies are loaded
+during git hook execution.
+
```

Comment:
Please add a short note on which mode is the default if the mode is not explicitly selected. I know it is explained below in line 137, but having it here would make it easier for readers to decide whether they need/want to adapt their configuration.